components/feature/app-links/src/main/java/mozilla/components/feature/app/links/AppLinksUseCases.kt

```diff
@@ -17,6 +17,9 @@ import java.util.UUID
 private const val EXTRA_BROWSER_FALLBACK_URL = "browser_fallback_url"
 private const val MARKET_INTENT_URI_PACKAGE_PREFIX = "market://details?id="
 
+@VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+internal const val APP_LINKS_CACHE_INTERVAL = 30 * 1000L // 30 seconds
```

Comment:
No data backing up this choice for 30 seconds.  This could be shorter (5 seconds could work as well) since this is to avoid the performance hits that occurs when `BrowserMenu` checks for visibility of the menu item.  Any thoughts?